docs/paper/dev/getting-started/project-setup.mdx

```diff
@@ -187,6 +187,49 @@ If everything went well, you should see something like this:
 
 ![](assets/paper-plugin-overview.png)
 
+## Plugin remapping
+
+As of 1.20.5, Paper will be shipping with a Mojang-mapped runtime instead of reobfuscating the server to Spigot mappings.
+If you are using Spigot/Bukkit plugins, your plugin will automatically be remapped to Spigot mappings.
+This means that the server will have to deobfuscate and remap the plugin jar every time it is loaded for the first time.
+If you want to skip remapping, you need to add the following code to your `build.gradle(.kts)`.
+
+:::note[Paper plugins]
+
+if you are using paper plugins this step is not needed as plugins will be assumed to be Mojang-mapped.
+
+:::
+
+```groovy title="build.gradle(.kts)"
```

Comment:
all build scripts in the docs are afaik Kotlin DSL, so specifying the Groovy variant is unnecessary
```suggestion
```kotlin title="build.gradle.kts"
```